frontend/tet/shared/src/util/getFormattedDate.ts

```diff
@@ -0,0 +1,10 @@
+const getFormattedDate = (date: string): string => {
```

Comment:
I think that calling `convertToUIDateFormat(string) || ''` method from `date.utils.ts` (in shared code) does effectively the same. It's better to collect all date utils to the same shared file, as they are quite same accross all the projects.

Reply:
True, thanks!